README.md

```diff
@@ -227,51 +239,77 @@ Create Azure Monitor
 
 # Add App Insights extension
 az extension add -n application-insights
+az feature register --name AIWorkspacePreview --namespace microsoft.insights
+az provider register -n microsoft.insights
 
 # Create App Insights
-export He_AppInsights_Key=$(az monitor app-insights component create -g $He_App_RG -l $He_Location -a $He_Name --query instrumentationKey -o tsv)
+az monitor app-insights component create -g $He_App_RG -l $He_Location -a $He_Name -o table
 
 # add App Insights Key to Key Vault
-az keyvault secret set -o table --vault-name $He_Name --name "AppInsightsKey" --value $He_AppInsights_Key
+az keyvault secret set -o tsv --query name --vault-name $He_Name --name "AppInsightsKey" --value $(az monitor app-insights component show -g $He_App_RG -a $He_Name --query instrumentationKey -o tsv)
 
-# Optional: Run ./saveenv.sh to save latest variables
+# save the env variable - use eval $He_AppInsights_Key
+export He_AppInsights_Key='az keyvault secret show -o tsv --query value --vault-name $He_Name --name AppInsightsKey'
+
+# save the environment variables
+./saveenv.sh -y
 
 ```
 
-Deploy the container to App Service or AKS
+> WIP
 
-- Instructions for [App Service](docs/AppService.md)
-- Instructions for [AKS](docs/aks/README.md#L233) (currently requires csharp)
+Run the app locally
 
-Run the Integration Test
+Should we move this before setup ACR?
 
-```bash
+> End WIP
 
-# run the tests in the container
-docker run -it --rm retaildevcrew/webvalidate --host https://${He_Name}.azurewebsites.net --files baseline.json
+Deploy the container to App Service or AKS
+
+- Instructions for [App Service](docs/AppService.md)
+- Instructions for [AKS](docs/aks/README.md#L233)
 
-```
 ## Dashboard setup
 
 Replace the values in the `Helium_Dashboard.json` file surrounded by `%%` with the proper environment variables
-after making sure the proper environment variables are set (He_Sub, He_App_RG, Imdb_RB and Imdb_Name)
+after making sure the proper environment variables are set (He_Sub, He_App_RG, Imdb_RG and Imdb_Name)
 
 ```bash
 
-cd $REPO_ROOT/docs/dashboard
-sed -i "s/%%SUBSCRIPTION_GUID%%/${He_Sub}/g" Helium_Dashboard.json && \
+export He_Sub='az account show -o tsv --query id'
+
+### TODO - this won't work since the instructions clone the language repo, not the helium repo
+### One option is to make dashboard setup a separate md file and include instructions for cloning
+###   the helium repo
+### Another option is to add the dashboard files to each of the language repos
+
```

Comment:
We could use wget to download the file instead of cloning the whole repo, but feels a little strange

Reply:
let's discuss in standup